include/ignition/math/Helpers.hh

```diff
@@ -830,6 +831,131 @@ namespace ignition
       return output_string.str();
     }
 
+    /// \brief Convert a string to a std::chrono::steady_clock::time_point
+    /// \param[in] _timeString The string to convert in general format
+    /// "dd hh:mm:ss.nnn" where n is millisecond value
+    /// \return A std::chrono::steady_clock::time_point containing the
+    /// string's time value
+    inline std::chrono::steady_clock::time_point stringToTimePoint(
+        const std::string &_timeString)
+    {
+      std::chrono::steady_clock::time_point timePoint =
+        math::secNsecToTimePoint(-1, 0);
```

Comment:
I'm seeing an msbuild warning when compiling this from source for sdformat:

~~~
'argument': conversion from 'int' to 'const uint64_t', signed/unsigned mismatch
~~~

* https://build.osrfoundation.org/job/sdformat-ci-sdformat9-windows7-amd64/37/msbuild/#issuesContent

Reply:
I'm guessing it's complaining about the implicit cast of `-1` to `const uint64_t`

Reply:
Yes, I just realized this, I guess an invalid string will have to return 0 now, I was using -1 as an indication of error, but now it seems that's not going to be possible